libraries/security/src/main/java/org/datatransferproject/security/CryptoTransformations.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.datatransferproject.security;
+
+/** Provides compile-time constants for secure crypto transformations. For use 
+in creation of Encrypter/Decrypter */
+public final class CryptoTransformations {
```

Comment:
How about we use an enum to be typsafe then we can use it in the case conditions?

public enum CryptoTransformation {
    AES_CBC_NOPADDING("AES/CBC/NoPadding"),
    RSA_ECB_PKCS1("RSA/ECB/PKCS1Padding"),

    public final String algorithm;
 
    private CryptoTransformation(String algorithm) {
        this.algorithm = algorithm;
    }
}